tests/devapps/XamarinDev/XamarinDev/App.xaml.cs

```diff
@@ -14,7 +14,7 @@ public partial class App : Application
 
         public static object RootViewController { get; set; }
 #if !IS_APPCENTER_BUILD
-        public const string DefaultClientId = "4a1aa1d5-c567-49d0-ad0b-cd957a47f842"; // in msidentity-samples-testing tenant -> PublicClientSample
+        public const string DefaultClientId = "16dab2ba-145d-4b1b-8569-bf4b9aed4dc8"; // in msidentity-samples-testing tenant -> PublicClientSample
```

Comment:
>16dab2ba-145d-4b1b-8569-bf4b9aed4dc8" [](start = 47, length = 37)

is this okay to change? will everything still work for all the tests, including iOS? If not, maybe include a separate android broker id.

Reply:
not needed. will revert